ES/ESdd037.xml

```diff
@@ -82,28 +83,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <title ref="LIT4035SenkessarDL#Pagwemen"/>                   
                      </msItem>  
                      <colophon xml:id="coloph1" xml:lang="gez">
-                        <locus target="#164vc"/>ተፈጸመ᎓ መጽሐፈ᎓ ስንክሳር᎓ ለእግዚአብሔር᎓ በሥምረቱ᎓ እንዘ᎓
-                        ይረድአነ᎓ በክሂሎቱ። ስብሐት᎓ ሎቱ። እስመ᎓ ተለዓለ᎓ ስሙ᎓ ለባሕቲቱ። በ፸፻᎓፩፻᎓፵᎓ወ፬᎓ ዓመተ᎓ ምሕረት᎓ እንዘ᎓
-                        ዓበቅቴ᎓ ፲ወ፰። መጥቅዕሂ᎓ ፲ወ፪᎓ ጥንትዮን᎓ ፬ በሳድሳይ᎓ ወርኅ᎓ ለፀሐይ᎓ ጊዜ᎓ ሥርቀቱ። ለሳብዕ᎓ ወረኅ᎓ እምቅድመ᎓
-                        ግብአቱ᎓ እም፯ሠርቀ᎓ ሌሊት᎓ አመ፲ወ፭᎓ ለየካቲት᎓ ሠርቀ᎓ መዓልት᎓ ለንጉሥነ᎓ ፋሲለደስ᎓ በ፲ወ፱᎓ ዓመተ᎓ መንግሥቱ᎓
-                        ጳጳስሂ᎓ አባ᎓ ሚካኤል᎓ ወመምህርሂ᎓ አባ᎓ ዘሥላሴ᎓ ወቀይሰ᎓ ገበዝሂ᎓ ብእሴ᎓ እግዚአብሔር᎓ ወሱታፌሆሙ᎓ ገብርኤል᎓
-                        ለዘጸሐፎ᎓ ዘወልደ᎓ ማርያም᎓ ወለዘአጽሐፎ᎓ ለዘ᎓ አንበቦ᎓ ወለዘተርጐሞ᎓ ወለዘሰምዓ᎓ ቃላቲሁ᎓ ይጽሐፍ᎓ ስሞሙ᎓ ኀበ᎓
-                        ዓምደ᎓ ወርቅ᎓ በቀለመ᎓ ጽድቅ። ክርስቶስ᎓ ሊቅ᎓ አመ᎓ ይመጽእ᎓ በንጥረ᎓ መባርቅት᎓ ወድልቅል[ቅ]᎓ ወተፍጻሜቱኒ᎓ ኮነ᎓
-                        በዕለተ᎓ ሠሉስ᎓ በ፯ሰዓት።
+                        <locus target="#164vc"/> 
```

Comment:
In the new version, there are no spaces after word dividers in most of the colophon. Maybe there is some issue with the font you use in oxygen?